src/Sql/Predicate/In.php

```diff
@@ -122,9 +122,10 @@ public function getExpressionData()
             foreach ($values as $argument) {
                 list($replacements[], $types[]) = $this->normalizeArgument($argument, self::TYPE_VALUE);
             }
+            $values = count($values) > 0 ? array_fill(0, count($values), '%s') : [];
```

Comment:
Why this? `array_fill()` returns `[]` if `$values === []`. Please remove this change, the previous code is ok.